app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
```

Comment:
We don't need a create action here to create a new customer (although we might want it in the future!). I think what you were trying here is a create that the form in US 2 could direct to, but that forms helps us associate an item to a customer, so we actually want to route that to our joins controller (see comment above)